src/libs/collect.c

```diff
@@ -1028,7 +1028,7 @@ static GtkTreeModel *_create_filtered_model(GtkTreeModel *model, dt_lib_collect_
         // Check if this path also matches a filmroll
         gtk_tree_model_get(model, &iter, DT_LIB_COLLECT_COL_PATH, &pth, -1);
         DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
-                                    "SELECT id FROM main.film_rolls WHERE folder LIKE ?1", -1, &stmt, NULL);
+                                    "SELECT id FROM main.film_rolls WHERE folder = ?1", -1, &stmt, NULL);
```

Comment:
again, won't this break seach using patterns ?

Reply:
yes, plus case sensitivity ...